outputs.tf

```diff
@@ -1,8 +1,12 @@
-#output "ecs_cluster" {
-#  value = ["${aws_subnet.private.*.id}"]
-#}
-
 output "cluster_id" {
   value = "${aws_ecs_cluster.cluster.id}"
 }
-# ECS servers
+
+
+output "autoscaling_group" {
+  value = {
+    id   = "${aws_autoscaling_group.ecs.id}"
+    name = "${aws_autoscaling_group.ecs.name}"
+    asrn = "${aws_autoscaling_group.ecs.arn}"
```

Comment:
`asrn` => `arn`. I don't think this is good idea to output a map instead of plain string values as we do in most cases. Why do you use `map` here?